test/manipulation.test.js

```diff
@@ -901,6 +902,103 @@ describe('manipulation', function () {
         e.should.be.eql(new Error('Invalid date: X'));
       }
     });
+  });
+
+  describe('update|updateAll', function() {
+    beforeEach(function destroyFixtures(done) {
+      Person.destroyAll(done);
+    });
+
+    beforeEach(function createFixtures(done) {
+      var people = [{
+        name: 'foo',
+        age: 19
+      }, {
+        name: 'bar',
+        age: 20
+      }, {
+        name: 'baz',
+        age: 21
+      }, {
+        name: 'qux',
+        age: 22
+      }, {
+        name: 'quux',
+        age: 23
+      }];
```

Comment:
More importantly, you should use values that make it easy to distinguish between valid and invalid values, or any other important constraint used in the test. So for example if your test data uses english first names, and then you have `name: 'does-not-exist'` in your test, then it's immediately clear what's going on. On the other hand, if your test data has "foo" and "bar", then it's not immediately clear whether `name: 'baz'` is referring to an existing name or not.